spec/Conversions.tex

```diff
@@ -35,13 +32,25 @@ \section{Implicit Conversions}
       the assignment is converted to the type of the variable
       or another lvalue on the left-hand side of the assignment.
 
+\item In a variable or field declaration, the initializing expression
+      is converted to the type of the variable or field.
+      The initializing expression is the r.h.s. of the \chpl{=}
+      in the declaration, if present, or in the field initialization
+      statement in an initializer.
+
 \item The actual argument of a function call or an operator is converted
       to the type of the corresponding formal argument, if the formal's
       intent is \chpl{param}, \chpl{in}, \chpl{const in}, or an abstract intent
       (\rsec{Abstract_Intents}) with the semantics of
       \chpl{in} or \chpl{const in}.
 
-% MPF: This rule doesn't seem to be implemented right now,
+\item The actual type argument of a function call or an operator is converted
+      to the corresponding formal argument (\rsec{Implicit_Type_Arg_Conversions}).
+      In this case, the formal has the \chpl{type} intent, including
+      the \chpl{this} formal of a type method.
+      A special rule defines how the type is converted in this case.
```

Comment:
Let's replace the "A special rule" with `See \rsec{Implicit_Type_Arg_Conversions}` and maybe remove that reference from above.
Maybe you wanted to say that not all implicit conversions are available here, which would be better to say directly than to say there is a special rule.